1_5_3 and 1_5_4/includes/templates/YOUR_TEMPLATE/templates/tpl_modules_main_product_image.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
Umm.. I'm not seeing the significant change from ZC 1.5.4 to ZC 1.5.5 that would necessitate this being in a separate location for either version.  A change in the header doesn't necessarily make that significant enough to warrant a new file location... I do see that in ZC 1.5.5 the `language="javascript"` part of `<script language="javascript" type="text/javascript">` was finally removed as discussed in the forum that it is not necessary, but I'm not seeing why the version for ZC 1.5.5 couldn't actually have some "notes" applied to the header to indicate that it could be used on all four currently supported versions and not have to be in a separate directory by version for it.  In fact, when creating the 1.5.3/1.5.4 directory the overall affect was evaluated to determine if a separate 1.5.3 directory was required or if it would still work with the 1.5.4 changes for the particular file(s) applied...